js/src/lib/components/InferenceWidget/widgets/ImageSegmentationWidget/ImageSegmentationWidget.svelte

```diff
@@ -223,10 +228,17 @@
 		outputJson = "";
 	}
 
-	onMount(() => {
+	onMount(async () => {
```

Comment:
I think this is a pattern that should be avoided if possible? cc @pngwn @beurkinger 

(particularly when it returns something to be called on destroy, though this is not the case here)

Reply:
thanks for catching that! https://github.com/huggingface/hub-docs/pull/293/commits/de0396992e759e0f75eb617c5f1ef14682f0e56f fixes those instances